openstack/cronus/values.yaml

```diff
@@ -526,7 +526,7 @@ poller: # service that handles received emails
   statefulSet:
     enabled: false
   action: forwardEmail # supported values: forwardEmail, activateEmail, simulator
-  pollInterval: 5 # should be less than a default cronus ResponseHeaderTimeout (9 seconds)
+  pollInterval: 5s # should be less than a default cronus ResponseHeaderTimeout (9 seconds)
```

Comment:
quick question: as it is "enabled: false" shouldn't nothing be deployed?

Reply:
Ah, potentially it is enabled in the other env but as the wrong default

Reply:
Yes, the reason is to control the deployment from cc/secrets per region

Reply:
Hi David, yes I know about the per region :) but this still shouldn't have the wrong default in the root values file. And, I believe shouldn't even be in this values file, but in the poller chart values file (see comment on the PR for cronus-qa pipeline)